charcoal-autocomments/comments.js

```diff
@@ -3,35 +3,35 @@ callback(
   [
     {
       name: "[Q,A] Vandalism",
-      description: "Please do not vandalise your posts. Once you have submitted a post, you have licensed the content to the $SITENAME$ community at large ([under the CC BY-SA license](https://creativecommons.org/licenses/by-sa/3.0/)). By SE policy, any vandalism will be reverted."
+      description: "Please do not vandalize your posts. Once you have submitted a post, you have licensed the content to the $SITENAME$ community at large ([under the CC BY-SA license](https://creativecommons.org/licenses/by-sa/4.0/)). By SE policy, any vandalism will be reverted."
     },
     {
       name: "[Q,A] Rude",
-      description: "Please remember that we have a **[Be Nice](https://$SITEURL$/help/be-nice)** policy on $SITENAME$; this includes not using profanity or obscenities in your posts. In the future, please keep all posts profanity-free. In the meantime, I've edited this post to keep it in line with the policies. Thanks for understanding."
+      description: "Please remember that we have a **[Code of Conduct](https://$SITEURL$/conduct)** on $SITENAME$; this includes not using profanity or obscenities in your posts. In the future, please keep all posts profanity-free. In the meantime, I've edited this post to keep it in line with policies. Thanks for understanding."
```

Comment:
Explicitly calling it the code of conduct and using the main link is good.

Reminder for me later: if we're going to change URLs, then we might as well remove the "https://$SITEURL$", which is extraneous, given that it's an absolute path within the same domain.